bpf-gpl/nat.h

```diff
@@ -72,12 +72,14 @@ static CALI_BPF_INLINE int skb_nat_l4_csum_ipv4(struct __sk_buff *skb, size_t of
 	return ret;
 }
 
-static CALI_BPF_INLINE struct calico_nat_dest* calico_v4_nat_lookup2(__be32 ip_src,
+static CALI_BPF_INLINE struct calico_nat_dest* calico_v4_nat_lookup3(__be32 ip_src,
 								     __be32 ip_dst,
 								     __u8 ip_proto,
 								     __u16 dport,
 								     bool from_tun,
-								     nat_lookup_result *res)
+								     nat_lookup_result *res,
+								     bool affinity_always,
+								     bool affinity_to_update)
```

Comment:
`affinity_to_update` reads as "pointer to the affinity to update", I'd probabaly call this `update_affinity`